website/docs/r/ecs_instance_set.html.markdown

```diff
@@ -160,6 +161,13 @@ The `network_interfaces` supports the following:
 * `network_interface_name` - (Optional, ForceNew) The name of ENI.
 * `primary_ip_address` - (Optional, ForceNew) The primary private IP address of ENI.
 
+#### Block data_disks
+
+The `exclude_instance_filter` supports the following:
+
+* `key` - (Required) The type of the excluded. Valid values: `InstanceId`, `InstanceName`.
+* `value` - (Required) The value of the excluded.
+
```

Comment:
value： The identification of the excluded instances. It is a list of instance Ids or names.